migrate/20250307_add_location_index.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  no_transaction
+
+  up do
+    alter_table(:location) do
+      add_index [:project_id, :display_name], name: :location_project_id_display_name_uidx, unique: true, concurrently: true
```

Comment:
Considering the size of the `location` table, is it really necessary to build the index concurrently?  Can we disable the Rubocop warning and include this in the single migration?

Reply:
Makes sense, I moved it.